cmds/exp/getty/getty.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// getty Open a TTY and invoke a shell
+// There are no special options and no login support
+// Also getty exits after starting the shell so if one exits the shell, there
+// is no more shell!
+//
+// Synopsis:
+//     getty <port> <baud> [term]
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strconv"
+
+	"github.com/u-root/u-root/pkg/termios"
+	"github.com/u-root/u-root/pkg/upath"
+)
+
+var (
+	verbose = flag.Bool("v", false, "verbose log")
+	debug   = func(string, ...interface{}) {}
+	cmdList []string
+	envs    []string
+)
+
+func init() {
+	r := upath.UrootPath
+	cmdList = []string{
+		r("/bin/defaultsh"),
+		r("/bin/sh"),
+	}
+}
+
+func main() {
+	flag.Parse()
+
+	if *verbose {
+		debug = log.Printf
+	}
+
+	port := flag.Arg(0)
+	baud, err := strconv.Atoi(flag.Arg(1))
+	if err != nil {
+		baud = 0
+	}
+	term := flag.Arg(2)
+
+	ttyS, err := termios.NewTTYS(port)
+	if err != nil {
+		debug("Unable to open port %s: %v", port, err)
```

Comment:
I think this should be log.Fatal; you want to see this error, and then you can remove the returrn.

Reply:
done